File: Programming/Assignment/Module 3 (Fundamentales of Programming)/Basic Logic Programs/Loop Logic Programs/Factorial number.c
// WAP to print factorial of given number

#include<stdio.h>
main()
{
	int n, i, fac=1;
	
	printf("Enter Number : ");
	scanf("%d",&n);
	
	for(i=1;i<=n;i++)
	{
		fac*=i;	// Multiply fac by the current value of i
	}
	printf("Factorial is : %d",fac);
}

File: Programming/Assignment/Dart/Module 1/Factorial number.dart
import 'dart:io';

void main() {
  print("Enter a number : ");
  int number = int.parse(stdin.readLineSync()!);

  int fac = 1;

  for (int i = 1; i <= number; i++) {
    fac *= i; // Multiply factorial by i
  }

  print("The factorial of $number is: $fac");
}
